protocol/app/upgrades/v8.0/upgrade_test.go

```diff
@@ -0,0 +1,71 @@
+package v_8_0
+
+import (
+	"testing"
+
+	"cosmossdk.io/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+	accountplustypes "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/types"
+	"github.com/stretchr/testify/suite"
+)
+
+type UpgradeTestSuite struct {
+	suite.Suite
+
+	tApp *testapp.TestApp
+	Ctx  sdk.Context
+}
+
+func TestUpgradeTestSuite(t *testing.T) {
```

Comment:
For my understanding did you write these tests based on some existing template? Have we considered using the conventional upgrade containter test ([example](https://github.com/dydxprotocol/v4-chain/blob/ebc4a8efa93cb2dffe6734c1915829c889c51e9d/protocol/app/upgrades/v7.0/upgrade_container_test.go#L29)), which is more end-to-end?

The current test more like a unit test on `migrateAccountplusAccountState` - no upgrade is actually being run and we are just calling `migrateAccountplusAccountState` with whitebox set-up and postcheck. Probably more accurate to rename this test to `TestMigrateAccountplusAccountState`.